tests/lib/rules/id-length.js

```diff
@@ -564,6 +571,13 @@ ruleTester.run("id-length", rule, {
             errors: [
                 tooLongErrorPrivate
             ]
+        },
+        {
+            code: "var 𠮟 = 2",
+            parserOptions: { ecmaVersion: 6 },
+            errors: [
+                tooShortError
+            ]
```

Comment:
Can we cover more test cases here? [Playground Link ](https://eslint.org/play/#eyJ0ZXh0IjoiLyplc2xpbnQgaWQtbGVuZ3RoOiBcImVycm9yXCIqL1xuXG52YXIgbXlPYmogPSB7IPCQjJg6IDEgfTtcblxuKPCQjJgpID0+IHsg8JCMmCAqIPCQjJggfTtcblxuY2xhc3Mg8KCunyB7IH1cblxuY2xhc3MgRm9vIHsg8JCMmCgpIHt9IH1cblxuY2xhc3MgRm9vMSB7ICPwkIyYKCkge30gfVxuXG5jbGFzcyBGb28yIHsg8JCMmCA9IDEgfVxuXG5jbGFzcyBGb28zIHsgI/CQjJggPSAxIH1cblxuZnVuY3Rpb24gZm9vMSguLi7wkIyYKSB7IH1cblxuZnVuY3Rpb24gZm9vKFvwkIyYXSkgeyB9XG5cbnZhciBbIPCQjJggXSA9IGFycjtcblxudmFyIHsgcHJvcDogW/CQjJhdfSA9IHt9O1xuXG5mdW5jdGlvbiBmb28oe/CQjJh9KSB7IH1cblxudmFyIHsg8JCMmCB9ID0ge307XG5cbnZhciB7IHByb3A6IPCQjJh9ID0ge307XG5cbih7IHByb3A6IG9iai7wkIyYIH0gPSB7fSk7Iiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjoibGF0ZXN0Iiwic291cmNlVHlwZSI6InNjcmlwdCIsImVjbWFGZWF0dXJlcyI6e319LCJydWxlcyI6e30sImVudiI6e319fQ==)

```js
var myObj = { 𐌘: 1 };

(𐌘) => { 𐌘 * 𐌘 };

class 𠮟 { }

class Foo { 𐌘() {} }

class Foo1 { #𐌘() {} }

class Foo2 { 𐌘 = 1 }

class Foo3 { #𐌘 = 1 }

function foo1(...𐌘) { }

function foo([𐌘]) { }

var [ 𐌘 ] = arr;

var { prop: [𐌘]} = {};

function foo({𐌘}) { }

var { 𐌘 } = {};

var { prop: 𐌘} = {};

({ prop: obj.𐌘 } = {});
```

Reply:
Thank you! [39928ab](https://github.com/eslint/eslint/pull/16321/commits/39928ab89dafb4c77049e2256aa2ff10f3234928)